package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "dracula-wordpress",
+  "version": "1.0.0",
```

Comment:
Should we align this with the version that is in `dracula-wordpress.php`?

Reply:
Good catch.